includes/wikia/tasks/Tasks/ImageReviewTask.class.php

```diff
@@ -108,6 +108,75 @@ public function deleteFromQueue( Array $aDeletionList ) {
 		}
 	}
 
+	public function addToQueue() {
```

Comment:
I don't think it's that long in terms of complexity. The only part that makes it long in terms of line number is logging the result to Kibana at the end, so I don't think there's a logical split that really adds value (other than maybe getting the insert data array in a separate method). I removed some unnecessary variable assignments to make it a little more succinct, but let me know if you have a specific suggestion for splitting it. :)